tutorial.md

```diff
@@ -0,0 +1,305 @@
+<h1 align="center">
+  Astro Python SDK
+</h1>
+  <h3 align="center">
+  Tutorial<br><br>
+</h3>
+
+# Introduction
+
+This tutorial demonstrates how to use the Astro Python SDK through a simple ETL example that you can run on your local machine:
+* **Extract** a file from S3 into a Snowflake relational table
+* **Transform** that table in Snowflake
+* **Load** that transformed table into a reporting table in Snowflake
+
+You'll need to use existing Amazon S3 and Snowflake accounts, or create some trial versions.
+
+***
+# Let's do some setup
+
+## Install Airflow
+
+For macOS, you can follow these steps:
+* Install homebrew: 
+
+    `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+* `brew install pip`
+* `brew install python`
+* Install Airflow:
+
+    `pip3 install apache-airflow --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.2.5/constraints-3.9.txt"`
```

Comment:
@mikeshwe why are we encouraging the installation of Airflow with these constraints? To make sure the tutorial continues working? Next to the tutorial, would it make sense to have the `requirements.txt` in our own repo?

Reply:
for the constraints, if installing airflow works without specifying them, then I'll change to that. I thought I ran into some errors if I didn't specify them, but I could likely be mistaken. I'll try to test this out on another mac.

Also, I added a step below on adding `astro-projects` to the `requirements.txt` file

Reply:
i deleted this whole section on the pip installs I realized that you don't need them when you do `brew install astronomer/cloud/astrocloud

`